Dshop/apps/users/serializers.py

```diff
@@ -49,3 +56,53 @@ def validate(self, attrs):
             raise ValidationError(_('Unable to log in with provided credentials'))
         attrs['user'] = user
         return attrs
+
+
+class PasswordChangeSerializer(serializers.Serializer):
+    current_password = serializers.CharField(write_only=True)
+    new_password = serializers.CharField(write_only=True)
+    new_password_again = serializers.CharField(write_only=True)
+
+    def validate_current_password(self, value):
+        if not self.context['request'].user.check_password(value):
+            raise serializers.ValidationError(_('Current password does not match'))
+        return value
+
+    def validate(self, attrs):
+        if attrs['new_password'] and attrs['new_password_again']:
+            if attrs['new_password'] != attrs['new_password_again']:
+                raise ValidationError(_('Passwords does not match!'))
+            password_validation.validate_password(attrs['new_password'])
+        return attrs
+
+
+class UserDataReadSerializer(serializers.ModelSerializer):
+    email = serializers.EmailField(source='user.email')
+
+    class Meta:
+        model = CustomUser
+        fields = '__all__'
```

Comment:
Please, never use __all__
Please change it to list of fields.
Using this is dangerous - example - we will introduce PESEL or other personal information. And it ill automatically show in all the places. 
Please replace it with list of fields.